pg_chameleon/lib/sql_util.py

```diff
@@ -4,11 +4,26 @@
 
 
 def ci_string(s):
-    return string(s, transform=lambda x: x.upper())
+    """
+    This function creates a case-insensitive parser for a string
+
+    :param s: the string to make a case-insensitive parser for.
+    :return: case-insensitive parser for string s
+    :rtype: parsy.Parser
+    """
+    return string(s.upper(), transform=lambda x: x.upper()).result(s)
+
 
+def optional_space_around(p):
+    """
+    This function creates extends an existing parser with optional whitespace
```

Comment:
creates or extends?

Reply:
Ah sorry, should've done some proofreading before. This must be extends.